test/functional/kernel/reduce-basic/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+###############################################################################
+# Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+set(LOCTYPES Min2D Max2D Min2DView Max2DView Min2DViewTuple Max2DViewTuple)
+
+list(APPEND KERNEL_LOC_BACKENDS Sequential)
+
+if(RAJA_ENABLE_OPENMP)
+  list(APPEND KERNEL_LOC_BACKENDS OpenMP)
+endif()
+
+if(RAJA_ENABLE_TBB)
+  list(APPEND KERNEL_LOC_BACKENDS TBB)
+endif()
+
+
+#
```

Comment:
why are the tests broken apart into two different subsets, one for CPU back-ends and one for GPU back-ends? We should strive to drive each test code with all beck-ends.

Reply:
The non-tuple tests only work on the CPU backends because the GPU backends require a local tuple object for indexing. The ViewTuple tests can adequately perform 2D indexing on the device because they use a local RAJA::make_tuple.

I will try adding the omp-target loc-reduce tests to functional/forall/, but there are none existing which use RAJA::kernel.

Reply:
I suppose I can eliminate the custom 2D indexing used in the non-tuple CPU backend tests, or copy that object to the working resource.

Reply:
Please try to make the CPU and GPU tests as close to the same as possible. If there are differences, that indicates that we may have some work to do. Thanks.

Reply:
Turns out those tests were already working for GPU and OpenMPTarget . . .